README.md

```diff
@@ -62,6 +62,42 @@ running smoothly.
 An additional env variable `LOG_LEVEL` can be set to regulate which logs will be
 shown. Its value can be one of `trace`, `debug`, `info`, `warn` or `error`.
 
+## Testing with a local template playground
+
+In most cases this program will be tested with a local Carrot template
+playground. A few tips can be given for a good developer experience in this
+case. In particular:
+
+- Remember to start the playground before starting the answerer, as the answerer
+  needs a local RPC URL, and the playground bootstraps a local node as part of
+  its starting procedure.
+- Remember to update your local file with the correct local RPC URL depending on
+  the forked network on which the template is being set on.
+- In cases where a lot of past blocks must be indexed from the local RPC, the
+  local node's performance will be inevitably degraded, in some occasions even
+  rendering it unusable in the playground when simulating transactions. In this
+  case find out the current block number that the local node is pointing at
+  (also printed in the program's startup logs) and set it in the `checkpoints`
+  table in the database so that the program will detect the forked network to be
+  completely synced. Having a forked chain with id `CHAIN_ID`, a block number
+  `BLOCK_NUMBER` and the default connection string
+  `postgresql://user:password@127.0.0.1:5432/defillama-answerer` you can easily
+  update the checkpoint in the database by running
+  `psql postgresql://user:password@127.0.0.1:5432/defillama-answerer` in your
+  terminal followed by a
+  `INSERT INTO checkpoints (chain_id, block_number) VALUES (<CHAIN_ID>, <BLOCK_NUMBER>) ON CONFLICT (chain_id) DO UPDATE SET block_number = <BLOCK_NUMBER>;`
+  (remember the ending semicolon) in the Postgres prompt.
+- Local nodes such as Ganache work by default in "automining" mode, meaning that
+  no new block is produced unless a transaction is processed or unless manually
+  triggered. This is a problem because the answerer reacts on new block events
+  in order to process active oracles, so it might seem in certain cases that the
+  answerer has stopped processing events while in reality it's just waiting for
+  a new block to come. Luckily, most local nodes (such as Ganache and Anvil)
+  support the `evm_mine` RPC method, so in order to trigger a new block on a
+  local node running on a certain port `PORT` you can simply execute
+  `curl -X POST --data '{ "method": "evm_mine", "params": [] }' http:localhost:<PORT>`
```

Comment:
Missing `//` in `http:localhost:<PORT>`.

Reply:
Addressed in d053931a31d271a2b5796c6762ed101e0a71490d